buildscripts/build.sh

```diff
@@ -28,15 +28,15 @@ XC_OSS=(${XC_OS// / })
 
 # Delete the old dir
 echo "==> Removing old directory..."
-rm -rf bin/*
-mkdir -p bin/
+rm -rf bin/maya/*
```

Comment:
Will it be good to move this file to buildscripts/mayactl ?

Reply:
sure !!!